opencv.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: opencv
+  version: 4.11.0
+  epoch: 3
+  description: Open source computer vision and machine learning library
+  copyright:
+    - license: BSD-3-Clause
+
+data:
+  - name: libs
+    items:
+      calib3d: Camera calibration and 3D reconstruction.
+      core: Essential structures and functions.
+      dnn: Deep neural network inference.
+      features2d: Keypoint detection and description.
+      flann: Fast nearest neighbor searches.
+      highgui: Basic GUI and image display.
+      imgcodecs: Image file read/write.
+      imgproc: Core image processing routines.
+      ml: Traditional machine learning algorithms.
+      objdetect: Object detection methods.
+      photo: Image enhancement and denoising.
+      stitching: Panorama image stitching.
+      video: Video analysis and processing.
+      videoio: Video capture and output.
+
+environment:
+  environment:
+    CFLAGS: "$CFLAGS -g1"
+    CXXFLAGS: "$CXXFLAGS -g1"
+    CC: clang
+    CXX: clang++
+  contents:
+    packages:
+      - binutils
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang
+      - cmake
+      - eigen-dev
+      - glibc
+      - harfbuzz-dev
+      - hdf5-dev
+      - libtbb-dev
+      - lld
+      - ninja-build
+      - openblas-dev
+      - opencl-dev
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/opencv/opencv
+      expected-commit: 31b0eeea0b44b370fd0712312df4214d4ae1b158
+      tag: ${{package.version}}
+
+  - uses: cmake/configure
+    name: Cmake Configure
+    with:
+      output-dir: build
+      opts: |
+        -DENABLE_BUILD_HARDENING=ON \
+        -DENABLE_THIN_LTO=ON \
+        -DWITH_OPENCL=ON \
+        -DWITH_OPENGL=ON \
+        -DWITH_TBB=ON \
+        -DWITH_VULKAN=ON \
+        -DWITH_QT=ON \
+        -DWITH_ADE=OFF \
+        -DWITH_opencv_gapi=OFF \
```

Comment:
Probably, before anyone points in future, we can write a comment somewhere that Graph API volatile and we are disabling it 
https://docs.opencv.org/4.x/d0/d1e/gapi.html